src/modules/application/application.entity.ts

```diff
@@ -49,6 +49,10 @@ export class Application implements BaseEnsEntity {
   @Column()
   namespace: string;
 
+  @Field({ nullable: true })
+  @Column({ nullable: true })
```

Comment:
I think we need this to be `nullable` to start as some records won't have it populated. Or can we not have the `nullable` from the get go? 
```suggestion
```
Either way, we should populate existing records with a script (or maybe ENS sync will do it for us) and ensure that the column is not `nullable` in the long-term.

Reply:
Yeah, so we need to make it nullable from the start just because of the migration for adding the column else it fails because at that point they all have it as nullable. This is only temporary though as I plan to update it after it has been populated.

The ENS sync populates the namehash column for us. I can confirm that as I tested it locally